docs/pages/access-controls/guides/per-session-mfa.mdx

```diff
@@ -104,7 +104,7 @@ $ tctl create -f cap.yaml
 </Tabs>
 
 </TabItem>
-<TabItem scope="cloud" label="Teleport Enterprise Cloud">
+<TabItem scope="cloud" label="Teleport Enterprise">
```

Comment:
Same comment here re: tabs and dynamic resources instructions as the Locking and Passswordless guides.